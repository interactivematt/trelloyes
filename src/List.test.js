import React from 'react';
import ReactDOM from 'react-dom';
import List from './App';
import renderer from 'react-test-renderer';

describe('Card component', () => {
  it('renders in the UI', () => {
    const tree = renderer
    .create(<List header='First list' />)
    .toJSON()
    expect(tree).toMatchSnapshot();  
  })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
})