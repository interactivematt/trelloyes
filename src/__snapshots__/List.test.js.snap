// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders in the UI 1`] = `
<main
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
      Trelloyes!
    </h1>
  </header>
  <div
    className="App-list"
  />
</main>
`;
